pkg/probe/http/http.go

```diff
@@ -101,6 +101,14 @@ func DoHTTPProbe(url *url.URL, headers http.Header, client GetHTTPInterface) (pr
 		v := version.Get()
 		headers.Set("User-Agent", fmt.Sprintf("kube-probe/%s.%s", v.Major, v.Minor))
 	}
+	if _, ok := headers["Accept"]; !ok {
+		// Accept header was not defined. accept all
+		headers.Set("Accept", "*/*")
+	}
+	if headers.Get("Accept") == "" {
```

Comment:
this is just a theoretical possibility, correct?

Reply:
oh, taking it back. Why do you think we need to override the explicit setting of Accept header to the empty value?

Reply:
is it intended as a way to explicitly delete the header from the yaml?

Reply:
Yes, my idea was to apply the same logic as with the User-Agent and allow the user to explicitly remove the header if he wants to